portfolio/src/main/webapp/script.js

```diff
@@ -79,3 +80,19 @@ function getTimeStamp(comment) {
     return years === 1 ? years + " year ago" : years + " years ago";
   }
 }
+
+function renderForm() {
+  var commentForm = document.getElementById("comment-form");
+  const authDiv = document.getElementById("auth-message");
+  fetch("/auth").then(response => response.json())
+  .then(res => {
+    if (res[0]) {
+      commentForm.style.display = "block";
+      authDiv.innerHTML = "<p>Logout <a href=\"" + res[1] + "\">here</a>.</p>"
+    }
+    else {
+      commentForm.style.display = "none";
```

Comment:
Nice job hiding the comment form when the user isn't logged in.

Reply:
Thanks!